tests/torch/pruning/filter_pruning/test_algo.py

```diff
@@ -340,7 +338,7 @@ def test_calculation_of_flops(all_weights, pruning_flops_target, ref_flops):
 
     assert pruning_algo.current_flops == ref_flops
     # pylint:disable=protected-access
-    assert pruning_algo._calculate_flops_pruned_model_by_masks() == ref_flops
+    assert pruning_algo._calculate_flops_and_weights_pruned_model_by_masks()[0] == ref_flops
```

Comment:
Add check of weights with reference number
```
assert pruning_algo._calculate_flops_and_weights_pruned_model_by_masks()[1] == ref_weights
```

Reply:
Done